src/org/usfirst/frc/team2706/robot/commands/teleop/TeleopPneumaticControl.java

```diff
@@ -1,13 +1,16 @@
 package org.usfirst.frc.team2706.robot.commands.teleop;
 
 import org.usfirst.frc.team2706.robot.Robot;
+import org.usfirst.frc.team2706.robot.commands.autonomous.movements.StraightDriveWithEncoders;
 
 import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj.buttons.JoystickButton;
 import edu.wpi.first.wpilibj.command.Command;
 
 public class TeleopPneumaticControl extends Command {
 
 	@SuppressWarnings("unused")
+	private GenericHID driverJoystick = Robot.oi.getDriverJoystick();
 	private GenericHID joystick = Robot.oi.getOperatorJoystick();
```

Comment:
If you are going to have both a driver and operator joystick, could you rename to `operatorJoystick`?